### DISCUSSION TITLE
Upgrade after i flash the UBI firmware?

### DISCUSSION THREAD

**Discussion by bodyscape** [dangowrt/owrt-ubi-installer#121] [Q&A] [answered]
Hi,

After i flash the UBI firmware 22.03.1 can i upgrade with this bin file  : **https://downloads.openwrt.org/releases/22.03.2/targets/mediatek/mt7622/openwrt-22.03.2-mediatek-mt7622-linksys_e8450-squashfs-sysupgrade.bin**. 

I have a warning if i try to do it.
![image](https://user-images.githubusercontent.com/30115568/196724880-8769a7da-e9d0-4b33-ab8e-c75a61a8ee92.png)

or must wait a new **openwrt-22.03.2-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb** to be release?

Thank you

**Comment by dangowrt** [comment, marked as answer]
You have to use the file intended for the device when using UBI layout, ie.
[`openwrt-22.03.2-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb`](https://downloads.openwrt.org/releases/22.03.2/targets/mediatek/mt7622/openwrt-22.03.2-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb)

**Reply by bodyscape** [reply]
Oh thank you for the link, i will know where to check for the next version. :)
